src/help/zaphelp/contents/ui/dialogs/options/view.html

```diff
@@ -15,7 +15,34 @@
 If ZAP processes images.
 
 <H3>Display</H3>
-The layout of the 3 main windows.
+The layout of the 3 main panels.<br>
+The following options are available:
+<ul>
+    <li>Maximise left (Sites) tab - The 'tree' panel containing the Sites tab extends
+        for the full length of the left hand side. This will reduce the amount of
+        space available to the 'information' panel.</li>
+    <li>Maximise bottom (History etc) tabs - The 'information' panel extends for the
+        full length of the bottom. This will reduce the amount of space available to
+        the 'tree' panel.</li>
+    <li>Full Layout - The selected tab takes up the full screen. This is
+        useful when using ZAP on small screens.</li>
+</ul>
+
+<H3>Response Panel Position</H3>
+Allows to configure the position of the Response tab with respect to Request tab.<br>
+The following options are available:
+<ul>
+    <li>Tabs Side by Side - The Request and Response tabs are side by side. This
+        increases the information that can be displayed but means you cannot see both the
+        request and response at the same time.</li>
+    <li>Panels Side by Side - The Request panel is shown to the left of the Response
+        panel. This decreases the information that can be displayed but means you can see both
+        the request and response at the same time.</li>
+    <li>Request Shown Above Response - The Request panel is shown above the Response
+        panel. This decreases the information that can be displayed but means you can see both
+        the request and response at the same time.</li>
+</ul>
+This option does not apply when the Display option is set to Full Layout.
```

Comment:
"This option" meaning the Response Panel Position? Perhaps:

> The Response Panel Position option does not apply when the Display option is set to Full Layout.

would make things slightly more clear?